nncf/quantization/algorithms/bias_correction/torch_fx_backend.py

```diff
@@ -0,0 +1,121 @@
+    ) -> PTModelExtractionCommand:
+        return PTModelExtractionCommand([inp_id[0] for inp_id in input_ids], [out_id[0] for out_id in output_ids])
+
+    @staticmethod
+    def output_insertion_command(nncf_graph: NNCFGraph, target_point: PTTargetPoint) -> FXApplyTransformationCommand:
+        return FXApplyTransformationCommand(output_insertion_transformation_builder(target_point))
+
+    @staticmethod
+    def mean_statistic_collector(
+        channel_axis: int,
+        inplace: bool,
+        num_samples: Optional[int] = None,
+        window_size: Optional[int] = None,
+    ) -> TensorCollector:
+        return get_mean_statistic_collector(num_samples, channel_axis, window_size)
+
+    @staticmethod
+    def raw_statistic_collector(num_samples: Optional[int] = None) -> TensorCollector:
+        return get_raw_stat_collector(num_samples)
+
+    @staticmethod
+    def process_model_output(raw_data: Dict, output_name: int) -> Tensor:
+        return Tensor(raw_data[output_name])
+
+    @staticmethod
+    def get_activation_port_id(node: NNCFNode, nncf_graph: NNCFGraph) -> int:
+        return 0
+
+    @staticmethod
+    def get_bias_value(node: NNCFNode, model: torch.fx.GraphModule, nncf_graph: NNCFGraph) -> Tensor:
+        return Tensor(get_bias_value(node, nncf_graph, model))
+
+    @staticmethod
+    def get_input_name(model: torch.fx.GraphModule, node_name: str, input_port_id: int) -> str:
+        graph_node = get_graph_node_by_name(model.graph, node_name)
+        return graph_node.all_input_nodes[input_port_id].name
+
+    @staticmethod
+    def get_output_name(model: torch.fx.GraphModule, node_name: str, output_port_id: int) -> int:
+        graph_node = get_graph_node_by_name(model.graph, node_name)
+        nodes = list(graph_node.users)
+        while nodes:
+            node = nodes.pop()
+            if node.op == "call_function" and node.target == torch.ops.aten.clone.default:
+                nodes = list(node.users)
+                graph_node = node
+            elif node.op == "output":
+                return node.all_input_nodes.index(graph_node)
+
+        raise nncf.InternalError(f"Node with name {node_name} expected to have an output," " no outputs were found.")
+
+    @staticmethod
+    def is_quantized_weights(node: NNCFNode, nncf_graph: NNCFGraph) -> bool:
+        return is_quantized_weights(node, nncf_graph)
+
+    @staticmethod
+    def is_node_with_bias(node: NNCFNode, nncf_graph: NNCFGraph) -> bool:
+        return is_node_with_bias(node, nncf_graph)
+
+    @staticmethod
+    def remove_fq_from_inputs(model: torch.fx.GraphModule, nncf_graph: NNCFGraph) -> torch.fx.GraphModule:
+        return remove_fq_from_inputs(model, nncf_graph)
+
+    @staticmethod
+    def get_port_id(target_point: PTTargetPoint) -> int:
+        if target_point.target_type == TargetType.OPERATOR_POST_HOOK:
+            # Return 0 as default value for post hook port id.
```

Comment:
What is wrong with the `target_point.input_port_id` in the case of the `OPERATOR_POST_HOOK`?